CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 2.1.4
+ - feat: Adds detailed reporting for schema mismatches when checking database consistency.
+ - fix: Takes current transaction into account for include queries.
+ - fix: Loads passwords from env variables even if the `passwords.yaml` file doesn't exist.
+ - fix: Corrects type mismatch in `onTimeout` callback for cancelled subscriptions in `MethodStreamManager.closeAllStreams` method.
+ - fix: Correctly returns HTTP 400 error code if parameters passed to Serverpod are incorrect.
```

Comment:
Suggestion: Add a `fix` entry for https://github.com/serverpod/serverpod/commit/830108fec6354d2625e2c6bb63556ecb4a76b2d0

Since this was community reported and initially community fixed I think this would be nice to include.

@hampuslavin - Could you help phrase that fix so that it is understandable?

Reply:
Not sure I'm following: It's in the changelog?
> - fix: Corrects type mismatch in `onTimeout` callback for cancelled subscriptions in `MethodStreamManager.closeAllStreams` method.

Reply:
Oh, you are right and I'm blind! Sorry! 😓 

 Then we are good to go! 👍